CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Revision History for Microkit
 
+## Release 1.3.0-dev
+
```

Comment:
This part should probably be in a separate commit, because this is the commit we're going to tag as release (so we don't want the upcoming `-dev`  yet)

Reply:
Ah okay will split it up into a separate commit.

Reply:
Done, made the change to `VERSION.md` as well.